lib/experimental/fonts/font-library/class-wp-font-library.php

```diff
@@ -62,11 +62,17 @@ public static function get_expected_font_mime_types_per_php_version( $php_versio
 	 * @return WP_Font_Collection|WP_Error A font collection is it was registered successfully and a WP_Error otherwise.
 	 */
 	public static function register_font_collection( $config ) {
```

Comment:
For both `register_font_collection` and `WP_Font_Collection::__construct`, I think using $slug as the first arg, and then an array of config args second, would be closer to how other `register_` functions tend to work (register_custom_post_type, for example).